docs/probes.md

```diff
@@ -128,3 +122,15 @@ kubectl virt start probe-test
 kubectl get events --watch
 ```
 
+5. Check if the READY field is True, it may take a bit
+
+```sh
+kubectl get vms -w
+```
+
+6. (optional) Log in to the VM and watch the incoming qemu-ga commands
+
+```sh
+virtctl console readiness-probe
+journalctl -f
```

Comment:
Following the same thinking as `--type` and `--follow`, I have also changed the `kubectl apply --f /tmp/probe-test.yaml` line by `kubectl apply --filename /tmp/probe-test.yaml`. It may be less necessary than the ones mentioned. Nevertheless, I think it could be necessary for the sake of consistency. Do you agree?